### PR TITLE
Show information about new generated test262 excludelist

### PR DESCRIPTION
**Pull request by gergocs**
After update the excludelist with make_excludelist.py it doesn't gave
information about failing and new passing tests. This patch will fix this.

Signed-off-by: Gergo Csizi gergocs@inf.u-szeged.hu

### CODE REVIEW

**Review by ksh8281 — APPROVED** [review]
Such a convenient new feature!

File: tools/test/test262/make_excludelist.py [hunk 1]
```diff
@@ -30,6 +31,7 @@
 PROJECT_SOURCE_DIR = dirname(dirname(dirname(dirname(abspath(__file__)))))
 SCRIPT_SOURCE_DIR = dirname(abspath(__file__))
 DEFAULT_ESCARGOT = join(PROJECT_SOURCE_DIR, 'escargot')
+DEFAULT_EXCLUDE_LIST = "./tools/test/test262/excludelist.orig.xml"
```
Comment (clover2123) [inline tools/test/test262/make_excludelist.py:34]: How about defining this default file path as absolute path using `SCRIPT_SOURCE_DIR` defined above?

File: tools/test/test262/make_excludelist.py [hunk 2]
```diff
@@ -86,13 +88,15 @@ def main():
                         help='path to the engine to be tested (default: %(default)s)')
     parser.add_argument('--arch', metavar='NAME', choices=['x86', 'x86_64'], default='x86_64',
                         help='architecture the engine was built for (%(choices)s; default: %(default)s)')
+    parser.add_argument('--excludelist', metavar='PATH', default=DEFAULT_EXCLUDE_LIST,
+                        help='path to the exclude list (default: %(default)s)')
```
Comment (clover2123) [inline tools/test/test262/make_excludelist.py:92]: Why did you add this argument? Is this argument really used?
Currently, it looks that the fixed file (DEFAULT_EXCLUDE_LIST) is used only.
`args.excludelist` is nowhere used.

Comment (gergocs) [reply tools/test/test262/make_excludelist.py:92]: In line `131` and `132` It's used, but on better thought i think you're right it's not necessary, so i will delete it.

**Comment by gergocs** [comment]
Patch is ready for review

**Review by clover2123 — APPROVED** [review]
LGTM
